### PR TITLE
Use new repo path attributes

### PR DESCRIPTION
**Pull request by nik9000**
Mostly uses `{repname-root}` instead of `../../reponame` to find the
root of repositories. Because `../../` is confusing.

### CODE REVIEW

**Comment by nik9000** [comment]
Another one for you @lcawl ! This one's diff should be empty.

File: docs/en/getting-started/index.asciidoc [hunk 1]
```diff
@@ -11,11 +11,8 @@
 :blog-ref:          https://www.elastic.co/blog/
 :wikipedia:         https://en.wikipedia.org/wiki
 
-:kib-repo-dir:      {docdir}/../../../../kibana/docs
```
Comment (lcawl) [inline docs/en/getting-started/index.asciidoc]: Were these attributes intentionally removed?  I would expect them to just be updated with {stack-docs-root} paths

File: docs/en/glossary/index.asciidoc [hunk 2]
```diff
@@ -6,10 +6,10 @@
 :logstash-terms:       true
 :xpack-terms:          true
 
-include::{asciidoc-dir}/../../shared/versions/stack/{source_branch}.asciidoc[]
-include::{asciidoc-dir}/../../shared/attributes.asciidoc[]
+include::{docs-root}/shared/versions/stack/{source_branch}.asciidoc[]
+include::{docs-root}/shared/attributes.asciidoc[]
 
-:es-repo-dir:          {docdir}/../../../../elasticsearch/docs/reference
+:es-glossary:          {elasticsearch-reference}/glossary.asciidoc
```
Comment (lcawl) [inline docs/en/glossary/index.asciidoc]: I think in this context it makes me rethink the name of this attribute. Is it too late to rename it to something like "es-ref-root" or "es-ref-dir" or "es-reference-dir"? Then it's clearer that it's referring to a path, not to the book or its URL
```suggestion
:es-glossary:          {elasticsearch-reference}/glossary.asciidoc
```

Comment (nik9000) [reply docs/en/glossary/index.asciidoc]: https://github.com/elastic/docs/pull/1316

File: docs/en/getting-started/index.asciidoc (hunk 1 above)
Comment (nik9000) [reply docs/en/getting-started/index.asciidoc]: I switched everything to `{kibana-root}/docs`. I didn't think we needed the extra indirection there. Is that ok?

File: docs/en/glossary/index.asciidoc (hunk 2 above)
Comment (lcawl) [reply docs/en/glossary/index.asciidoc]: Thanks!

**Comment by nik9000** [comment]
Woah! I think this one isn't going to happen. It's been years!

**Pull request closed without merging**